run_tests.sh

```diff
@@ -51,23 +36,21 @@ testrepo () {
   MODPATHS=". $MODPATHS"
   for module in $MODPATHS; do
     echo "==> ${module}"
-    (cd $module && env GORACE='halt_on_error=1' CC=gcc $GO test -race \
+    (cd $module && env GORACE='halt_on_error=1' $GO test -race \
 	  ./...)
   done
 
   # check linters
-  if [[ $GOVERSION != 1.10 ]]; then
-    # linters do not work with modules yet
-    golangci-lint run --disable-all --deadline=10m \
-      --enable=gofmt \
-      --enable=vet \
-      --enable=gosimple \
-      --enable=unconvert \
-      --enable=ineffassign
-    if [ $? != 0 ]; then
-      echo 'golangci-lint has some complaints'
-      exit 1
-    fi
+  # linters do not work with modules yet
+  golangci-lint run --disable-all --deadline=10m \
+    --enable=gofmt \
+    --enable=vet \
+    --enable=gosimple \
+    --enable=unconvert \
+    --enable=ineffassign
+  if [ $? != 0 ]; then
+    echo 'golangci-lint has some complaints'
```

Comment:
With `set -ex`, I'm not sure if this message will hit.  What I did in lint.sh in dcrdata is like:

```sh
set +e
ERROR=0
trap 'ERROR=$?' ERR

for MODPATH in $MODPATHS; do
    pushd $MODPATH > /dev/null
    MOD=$(go list -m)
    echo "Linting:" $MOD
    superlint
    popd > /dev/null
done

popd > /dev/null # ROOT

if [ $ERROR -ne 0 ]; then
    exit $ERROR
fi
```

There are no submodules in dcrseeder, but note that if you want to get the full lint result when looping over modules, you pretty much have to switch back with `set +x`.